plugins/Sidebar/SidebarPlugin.py

```diff
@@ -562,8 +562,11 @@ def downloadGeoLiteDb(self, db_path):
         self.log.info("Downloading GeoLite2 City database...")
         self.cmd("progress", ["geolite-info", _["Downloading GeoLite2 City database (one time only, ~20MB)..."], 0])
         db_urls = [
-            "https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz",
-            "https://raw.githubusercontent.com/texnikru/GeoLite2-Database/master/GeoLite2-City.mmdb.gz"
+            "http://127.0.0.1:43110/1UPDatEDxnvHDo7TXvq6AEBARfNkyfxsp/GeoLite2-City.mmdb.gz",
```

Comment:
If anyone use a different zeronet ip/port,this won't work

Reply:
Yeah it would be more reliable if we would download it internally and not by the web interface, because it can be password protected / limit in other way

Reply:
I think that is easy.But just need to read the source code to find out,which variable to use.

Reply:
Sorry but I can't implement it.
You can merge the pull request then implement it.
Or I can add @shortcutme as writer on my fork

Reply:
@HelloZeroNet 
Please add Geolite db to ZeroUpdate

Reply:
@shortcutme added you as writer on ZeroNetPlus/ZeroNet
Now you can commit to my pull requests on the related branches

Reply:
@HelloZeroNet @shortcutme 
Invited you to the repo of this PR: https://github.com/ZeroNetPlus/ZeroNet/invitations

What I think is needed:
- [ ] Add Geolite db to ZeroUpdate
- [ ] IP/Port string instead of fixed values (you can explain me how to do it)
- [ ] When offline, instead of get download erro message, skip to the next download source

After it, this pull request will be ready to be merged.